quinzena1/portfolio/index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    meta charset="UTF-8">
```

Comment:
Faltou um `<` no início, o que fez com que esse `meta charset="UTF-8">` aparecesse na página.